src/fides/api/ops/models/privacy_request.py

```diff
@@ -851,6 +851,8 @@ class Consent(Base):
     data_use = Column(String, nullable=False)
     data_use_description = Column(String)
     opt_in = Column(Boolean, nullable=False)
+    has_gpc_flag = Column(Boolean, server_default="f", default=False, nullable=False)
+    conflicts_with_gpc = Column(Boolean, server_default="f", default=False, nullable=False)
```

Comment:
Same question on `server_default` of `f`. The answer in the other section will answer this on also.

Reply:
same as above